src/disk.hpp

```diff
@@ -292,13 +292,14 @@ struct BufferedDisk : Disk
         }
         else {
             // ideally this won't happen
-            std::cout << "Disk read position regressed. It's optimized for forward scans. Performance may suffer\n"
+            /* std::cout << "Disk read position regressed. It's optimized for forward scans. "
```

Comment:
In my testing, it will appear every time if the parallel read is applied. Since we already have the switch of parallel option, we may not need this warning anymore.

Reply:
would you mind setting a breakpoint to understand where that call is coming from?
the `BufferedDisk` is primarily meant to be used during *plotting*, and this warning is relevant. If it's printed, it suggest something changed to cause reading from the file to happen in reverse order.

The files used to read the qualities all use `std::fstream`, as far as I can tell.

Reply:
I see. This warning only appears in the Github ations check. That might not be related to parallel read changes. Thanks for informing me!